index.html

```diff
@@ -40,6 +42,30 @@ <h3>I’am a Software Developer</h3>
                 <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
             </ul>
         </section>
+        <section class="graphics">
+                <div class="graph-iten-one">
+                    <img src="./images/corner-img.svg" alt="" >
+                </div>
+                <div class="top-black">
+                    <img src="./images/top-black.svg" alt="">
+                </div>
+                <div class="top-white">
+                    <img src="./images/top-white.svg" alt=""> <span><img src="./images/top-small.svg" alt="" id="overlay-black"></span>
+                </div>
+                <img src="./images/curve-img.svg" alt="" id="curve-img">
+                <div class="graph-big-img-two">
+                    <img src="./images/big-img-two.svg" alt="" class="two-svg">
+                </div>
+               
+                <div class="graph-big-img">
+                    <img src="./images/big-img.svg" alt="" ><span><img src="./images/second-small.svg" alt="" id="overlay-big"></span>
+                </div>
+                <img src="./images/down-one.svg" alt="" class="down-img">
+                <img src="./images/down-two.svg" alt="" class="down-img-two">
+                <img src="./images/down-inner.svg" alt="" class="down-inner">
+                <img src="./images/down-bottom.svg" alt="" class="down-bottom">
+             
+        </section>
```

Comment:
- [ ] As the first reviewer requested, kindly remove this section and instead set the background image using CSS for the Headline section.

Reply:
Hi, every other adjustment has been made. On this part, kindly explained better because the first reviewer asked me to add the images because my first PR broke and the pictures were not displaying. I will appreciate it if you can explain better so I can fix the issues. Thank you for your support.

Reply:
- Hello, As the first reviewer requested this whole section should be removed as it doesn't exist in the original design. I believe it's the section used for displaying the image that's why I requested to set the image as a background using CSS for the Headline section.

Reply:
Thank you for your quick response. Please I will appreciate as much much help you can point me to, to fix this very issue.
Thank you for your support.